2024/shorts/mandelbrot/german/sentence_translations.json

```
@@ -1,46 +1,46 @@
 [
  {
-  "translatedText": "Die Mandelbrot-Menge ist eines der bekanntesten Bilder in der Mathematik.",
+  "translatedText": "Die Mandelbrot-Menge ist eine der bekanntesten Abbildungen der Mathematik.",
   "input": "The Mandelbrot set is one of the most iconic images in all of math.",
   "time_range": [
    0.0,
    3.36
   ]
  },
  {
-  "translatedText": "Sie beginnen mit einer komplexen Zahl, c, und definieren dann rekursiv eine Folge komplexer Zahlen, wobei die Folge mit 0 beginnt und jeder neue Wert als Quadrat des vorherigen Werts plus c definiert wird.",
+  "translatedText": "Beginnen Sie mit einer komplexen Zahl, c, und betrachten eine rekursive Reihe von komplexen Zahlen, angefangen mit 0, wobei jeder neue Wert als Quadrat des vorherigen Wertes plus c definiert ist.",
   "input": "You start with some complex number, c, and then you recursively define a sequence of complex numbers where the sequence starts with 0, and each new value is defined to be the square of the previous value, plus c.",
   "time_range": [
    3.78,
    16.28
   ]
  },
  {
-  "translatedText": "So nehmen Sie zum Beispiel bei der allerersten Iteration 0 zum Quadrat plus c, was bedeutet, dass z1 einfach c ist, und dann nehmen Sie für die nächste Iteration diese Zahl zum Quadrat plus c, was bedeutet, dass z2 das Quadrat von c plus c ist, und so weiter und so weiter.",
+  "translatedText": "So berechnet man bei der ersten Iteration 0 Quadrat und addiert c, also ist z1 gleich c. Für die nächste Iteration nimmt man dann diese Zahl ins Quadrat plus c, dass heißt c Quadrat plus c, und so weiter und so fort.",
```

```suggestion
  "translatedText": "So berechnet man bei der ersten Iteration 0 Quadrat und addiert c, also ist z1 gleich c. Für die nächste Iteration nimmt man dann diese Zahl quadriert plus c, das heißt c Quadrat plus c, und so weiter und so fort.",
```
Ins Quadrat erheben habe ich noch nie gehört, kann aber auch nur ich sein.
Duden schlägt [ins Quadrat erheben](https://www.duden.de/rechtschreibung/Quadrat#Bedeutung-2) vor,
Pons sagt [zum Quadrat](https://de.pons.com/%C3%BCbersetzung/englisch-deutsch/squared), finde aber alle möglichen Variationen.
Könnte man durch "quadriert" einfach vermeiden